src/main/java/uk/gov/hmcts/reform/prl/models/dto/ccd/CaseData.java

```diff
@@ -616,9 +615,13 @@ public static String[] temporaryFields() {
     //private final List<String> otherEmailAddress;
     private final String childrenList;
 
-    @JsonUnwrapped
-    @Builder.Default
-    private final ManageOrders manageOrders = ManageOrders.builder().build();
+    //    @JsonUnwrapped
+    //    @Builder.Default
+    //    private final ManageOrders manageOrders = ManageOrders.builder().build();
+
+    private final FL404b fl404bCustomFields;
```

Comment:
Can we add this property in ManageOrders.java? we can keep app manage order related attribute in one class, so that the size of case data attribute will not grow much going forward.